### PR TITLE
DOC-1317 - RS: CLI Utitilites Reorg rladmin help and rladmin info commands

### PR DESCRIPTION
**Pull request by cmilesb**
rladmin help: https://docs.redis.com/staging/jira-doc-1317/rs/references/cli-utilities/rladmin/help/
rladmin info: https://docs.redis.com/staging/jira-doc-1317/rs/references/cli-utilities/rladmin/info/

### CODE REVIEW

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+`rladmin info db` lists configurable settings for all databases.
+
+`rladmin info db <db:id | name>` lists configurable settings for the specified database.
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| db:id     | database id of the specified database |
+| name      | database name of the specified database |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:28]:
```suggestion
| db:id     | ID of the specified database (optional) |
| name      | Name of the specified database (optional) |
```

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    mtls_allow_outdated_certs: disabled
+    data_internode_encryption: disabled
+    proxy_policy: single
+    db_conns_auditing: disabled
+```
+
+## `info proxy`
+
+`rladmin info proxy` lists configurable settings for the proxy.
+
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:80]: Missing `rladmin info proxy { <id> | all }` syntax and the parameters table.

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+`rladmin info node` lists configurable settings for all nodes.
+
+`rladmin info node <id>` lists configurable settings for a single node.
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| id     | node id of the specified node |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:174]:
```suggestion
| id     | ID of the specified node |
```

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
+---
+Title: rladmin info
+linkTitle: info
+description: Lists configurable settings for specified databases, proxies, clusters, or nodes.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:4]:
```suggestion
description: Shows the current configuration of a cluster, database, node, or proxy.
```

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin info` lists configurable settings for specified databases, proxies, clusters, or nodes.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:13]:
```suggestion
`rladmin info` shows the current configuration of specified databases, proxies, clusters, or nodes.
```

File: content/rs/references/cli-utilities/rladmin/help.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+| Parameter | Description |
+|-----------|-------------|
+| command   | rladmind command to display help |
+
+### Returns
+
+Returns a list of all the options and parameters associated with the rladmin command.
+
+If `[command]` is specified, returns a list of all the options and parameters associated with that rladmin command.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/help.md:29]:
```suggestion
Returns a list of available `rladmin` commands.

If `[command]` is specified, returns a list of all the options and parameters for that `rladmin` command.
```

File: content/rs/references/cli-utilities/rladmin/help.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+``` sh
+rladmin help [command]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| command   | rladmind command to display help |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/help.md:23]:
```suggestion
|  command   |  Display help for this rladmin command (optional)  |
```

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+
+`rladmin info cluster` lists configurable settings for the cluster.
+
+```sh
+rladmin info cluster
+```
+
+### Parameters
+
+None.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:115]:
```suggestion
None
```

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+
+```sh
+rladmin info db [ <id> ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| id     | node id of the specified node |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:174]:
```suggestion
|  id  | ID of the specified node  |
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
It's looking good so far. I left some suggestions. Also, maybe we should reorder the rladmin info commands alphabetically.

**Comment by nermiller** [comment]
LGTM with Rachel's suggestions

**Review by nermiller — APPROVED** [review]
LGTM with Rachel's suggestions.

File: content/rs/references/cli-utilities/rladmin/info.md (hunk 2 above)
Comment (cmilesb) [reply content/rs/references/cli-utilities/rladmin/info.md:97]: Fixed in [1f2429e](https://github.com/RedisLabs/redislabs-docs/pull/1887/commits/1f2429ea1f5084c471bd704a33f206640e2813ec)

File: content/rs/references/cli-utilities/rladmin/info.md (hunk 9 above)
Comment (cmilesb) [reply content/rs/references/cli-utilities/rladmin/info.md]: Fixed in [217ff69](https://github.com/RedisLabs/redislabs-docs/pull/1887/commits/217ff6923523a42662d82993b676c3c72d98cb9c)

File: content/rs/references/cli-utilities/rladmin/info.md [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+    cnm_http_port: 8080
+    cnm_https_port: 9443
+```
+
+## `info node`
+
+`rladmin info node` lists the current configuration for all nodes.
+
+```sh
+rladmin info db [ <id> ]
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/info.md:172]:
```suggestion
rladmin info node [ <id> ]
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Two last change requests:
1. Fix a typo in rladmin info node
2. Reorder the rladmin info command sections alphabetically: cluster, db, node, proxy.

**Review by rrelledge — APPROVED** [review]
Looks good to me!

**Pull request merged**